core/admin/class-maxi-dashboard.php

```diff
@@ -485,9 +485,14 @@ public function maxi_blocks_settings()
             $content .= '</div>'; // maxi-dashboard_main-content_accordion-item
             $content .= $this->generate_item_header(__('Fonts and files', 'maxi-blocks'), false);
 
+
+            $description = '<h4>'.__('Use bunny.net fonts', 'maxi-blocks').'</h4>';
+            $description .= '<p>'.__('Use bunny.net CDN to serve fonts. GDPR compliant analogue to Google fonts', 'maxi-blocks').'</p>';
```

Comment:
@ckp267 could you please check name/description for this in dashboard, maybe need to change or add anything.